cou/utils/juju_utils.py

```diff
@@ -198,6 +198,15 @@ def is_from_charm_store(self) -> bool:
         """
         return self.origin == "cs"
 
+    def get_units_to_run_action(self) -> list[Unit | SubordinateUnit]:
+        """Get the unit or subordinate units in the application to run action.
+
```

Comment:
I'm not sure of the purpose from reading this - what does it mean to run action?

Reply:
I think `get_units` is better, but it's kind of conflicting with `self.units`

Reply:
It's really confusing that we have get_units, units, subordinates, subordinate_units, and now get_units_to_run_action...

Reply:
👌🏼